aws.yaml

```diff
@@ -1,3 +1,49 @@
+- active: false
+  apps:
+    - app: chart-operator
+      version: 0.11.4
+    - app: coredns
+      version: 1.1.3
+      componentVersion: 1.6.5
+    - app: cert-exporter
+      version: 1.2.1
+    - app: kube-state-metrics
+      version: 1.0.0
+      componentVersion: 1.9.0
+    - app: metrics-server
+      version: 1.0.0
+      componentVersion: 0.3.3
+    - app: net-exporter
+      version: 1.5.1
+    - app: node-exporter
+      version: 1.2.0
+      componentVersion: 0.18.1
+    - app: cert-manager
+      version: 1.0.4
+      componentVersion: 0.9.0
+    - app: cluster-autoscaler
+      version: 1.1.4
```

Comment:
There is no tarball for this release. https://github.com/giantswarm/default-catalog Can you check the version?

This is very easy to miss and I think we should have an automated check. So we know the versions here actually exist in the catalog.

@puja108 @piontec WDYT? Should we have an issue for that?

Reply:
It was in a changelog but not released yet. I just publish a new 1.1.4 tag there, Thanks for the notice.

Reply:
Thanks I'd still like to have the automated check. But I think it can be done after these changes.